cmd/up/forwards.go

```diff
@@ -105,5 +118,98 @@ func (up *upContext) sshForwards(ctx context.Context) error {
 		return fmt.Errorf("failed to add entry to your SSH config file")
 	}
 
-	return up.Forwarder.Start(up.Pod.Name, up.Dev.Namespace)
+	err := up.Forwarder.Start(up.Pod.Name, up.Dev.Namespace)
+	if err != nil {
+		return err
+	}
+
+	if isNeededGlobalForwarder(up.Manifest.GlobalForward) {
+		up.GlobalForwarderStatus = make(chan error, 1)
+		go up.setGlobalForwardsIfRequiredLoop(ctx)
+	}
+
+	return nil
+}
+
+func (up *upContext) setGlobalForwardsIfRequiredLoop(ctx context.Context) {
+	ticker := time.NewTicker(time.Second)
+
+	for {
+		if !isNeededGlobalForwarder(up.Manifest.GlobalForward) {
```

Comment:
Should we check this condition outside the for loop? I think `up.Manifest.GlobalForward` changes over the time

Reply:
we need to check it periodically because maybe we were not able to open all forwarders (due to port collision)